ads/temedya.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Since this is a new file, can you specify 2020 here?

Reply:
Thanks your feedback. Edited. fyi.